workflow/scripts/generate_virologist_output.py

```diff
@@ -142,7 +142,7 @@
 for file in snakemake.input.pangolin:
 
     pang_call = open(file, "r")
-    table[file.split("/")[-1].split(".")[0]] = [[],[],[],[],[],[],[],[],[],[],[],[]]
+    table[file.split("/")[-1].split(".")[0]] = [[] for i in range(12)]
```

Comment:
This can be done even simpler with 
```suggestion
    table[file.split("/")[-1].split(".")[0]] = [[]] * 12
```